UnityProject/Assets/Scripts/Managers/AddressableManager/AddressableCatalogueManager.cs

```diff
@@ -245,6 +245,7 @@ public static List<string> GetCataloguePathStreamingAssets()
 		var pathss = Application.streamingAssetsPath + "/AddressableCatalogues";
 		var Directories = System.IO.Directory.GetDirectories(pathss);
 		var Catalogues = new List<string>();
+		var MultiCatalogues = new List<string>();
 		foreach (var Directorie in Directories)
```

Comment:
Since you are here, rename all the 'var' variables to be camelCase

Reply:
ok